src/bust.py

```diff
@@ -85,14 +85,15 @@ def skip_song(self) -> None:
             self.play_song_task.cancel()
 
     async def play(self, interaction: Interaction, skip_count: int = 0) -> None:
+        await interaction.response.defer(ephemeral=True)
```

Comment:
I do agree that conceptually direct slash command handling is outside the bust controller, as mentioned in my other comment I think it's sort of a side effect of us handling errors in the BustController directly at the interaction level that it breaks some encapsulation by deferring outside, responding to errors inside, then having to check for errors after it returns etc.

My intention was to make the bustcontroller functions guarantee that they 1) take an interaction not responded to in any way 2) upon return, guarantee that interaction has been "dealt with".

As mentioned in my other comment, I think the "correct" abstraction is to allow returning some error codes/throwing some exceptions and handle all messages upon return outside the controller. But this might be just complicating things for no reason in our case. lmk what you think